src/vs/editor/contrib/stickyScroll/browser/stickyScrollProvider.ts

```diff
@@ -183,9 +199,52 @@ class StickyOutlineElement {
 		}
 		return new StickyOutlineElement(
 			range,
-			children
+			children,
+			undefined
 		);
 	}
+
+	public static fromFoldingModel(foldingModel: FoldingModel): StickyOutlineElement {
+		const regions = foldingModel.getRegions();
+		const startIndexes = regions.getStartIndexes();
+		const endIndexes = regions.getEndIndexes();
+		if (startIndexes.length === 0) {
+			return new StickyOutlineElement(
+				new StickyRange(-1, -1),
+				[],
+				undefined
+			);
+		}
+		let range = undefined;
```

Comment:
Add type annotation (`StickyRange | undefined`) or move declaration into the for-loop